packages/documentation/src/stories/foundation/icons/search-icons.stories.ts

```diff
@@ -2,7 +2,7 @@ import { Meta, StoryObj } from '@storybook/web-components';
 
 const meta: Meta = {
   id: '5704bdc4-c5b5-45e6-b123-c54d01fce2f1',
-  title: 'Components/Icons/Search for Icons',
+  title: 'Foundations/Icons',
```

Comment:
I like "Search for Icons" because it is meaningful, having several pages with the same name seems confusing. What do you think?

Reply:
Yep I agree